src/plugin.js

```diff
@@ -218,6 +219,8 @@ class Flash extends Tech {
       this.lastSeekTarget_ = time;
       this.trigger('seeking');
       this.el_.vjs_setProperty('currentTime', time);
+      // set currentTime cache
+      this.player_.currentTime();
```

Comment:
we should trigger a manual `timeupdate` like we are doing in the manual time updated. We should try and be explicit in things that are changing instead of calling methods and relying on their side-effects.

Reply:
The problem is that the player is `scrubbing` so the cache will get used. https://github.com/videojs/video.js/blob/master/src/js/control-bar/time-controls/current-time-display.js#L48

The cache is only set when calling `currentTime` on the player

Reply:
It seems like we would need a `timeupdate` after this line https://github.com/videojs/video.js/blob/master/src/js/control-bar/progress-control/seek-bar.js#L238 to force the ui to not use the cache. Would that be a reasonable approach?

Reply:
Then I'm not sure I understand what we're trying to fix here. Seems like this issue exists in the html5 tech as well and is just core to how seeking currently works in video.js.

Reply:
The issue is that when you seek while paused, the current time ui will show the previous current time value

 e.g. pause at 0:06 -> seek to 0:20, time ui still shows 0:06 -> seek to 0:30,  time ui updates to 0:20

This does not happen in html5 tech, appears to be related to using manual timeupdates.

Reply:
I can see it happening with the html5 tech, though, it isn't as consistent as in flash.
So, it sounds like we should fix this in video.js. Hopefully, adding that `timeupdate` trigger in the seekbar where you mentioned will fix it.

Reply:
As per previous comments, closing this pull request and creating a new PR on video.js